pages/api/getData.js

```diff
@@ -0,0 +1,29 @@
+import fs from 'fs'
+import path from 'path'
+
+//
+// Fetch ALL topics from local Data If no API is needed
+//
+
+export function getLocalTopics() {
+  // const fullPath = path.join(process.cwd(), "pages", "api", "topics.json");
+  // const fileContents = fs.readFileSync(fullPath, "utf8");
+  // const localData = JSON.parse(fileContents).topics;
+
+  // return { localData };
+  return { localData: [] }
+}
+
+//
+// Fetch ALL topics from API when available
+//
+
+export async function getTopics(language) {
+  const res = await fetch(
+    `${process.env.NEXT_CONTENT_API}${language}/lj-new-child.json?date=${process.env.NEXT_PUBLIC_BUILD_DATE}`
```

Comment:
If lj-new-child was a parameter would we be able to make something reusable? Perhaps there is a framework for AEM requests?

Reply:
we could but it depends on how we implement the AEM structure

Reply:
Sounds good